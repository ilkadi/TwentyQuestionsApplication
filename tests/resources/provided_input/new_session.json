{
  "qa_list": [],
  "final_guess": ""
}