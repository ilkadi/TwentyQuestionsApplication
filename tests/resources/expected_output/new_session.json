{
  "qa_list": [
    {
      "Question N1?": ""
    }
  ],
  "final_guess": ""
}